zookeeper/core/component.py

```diff
@@ -612,6 +612,14 @@ def configure(
                             conf_field_value = subclass()
                             break
 
+            # If this isn't the case, then it's a user type error, but we don't
+            # throw here and instead let the run-time type-checking take care of
+            # it (which will provide a better error message).
+            if utils.is_component_instance(conf_field_value):
+                # Set the component parent so that field value inheritence will
```

Comment:
This thing that says `Line #L619` is not covered by the tests is broken, because if I add a `raise Exception` on this line locally, then the tests fail. So I don't know how this tool thinks the line isn't being run.

Reply:
I'm just going to merge without worrying about that.